charts/inception/values.yaml

```diff
@@ -0,0 +1,495 @@
+    assertingPartySSOUrl: ""
+
+  # PreAuthentication settings, not cumulative with OAuth2 and SAML2 authentication modes (see [dedicated INCEpTION documentation](https://inception-project.github.io/releases/33.2/docs/admin-guide.html#sect_security_authentication_preauth)).
+  preAuthentication:
+    # -- Enables the PreAuthentication feature.
+    # @section -- INCEpTION authentication settings
+    enabled: false
+    # -- Name of the header holding the remote username.
+    # @section -- INCEpTION authentication settings
+    headerPrincipal: ""
+    # -- Role given to new pre-authenticated users.
+    # @section -- INCEpTION authentication settings
+    newUserRole: ""
+    # -- Roles to map to dedicated pre-authenticated users, in the `username: role` format.
+    # @section -- INCEpTION authentication settings
+    customUserRoles: {}
+
+# -- Extra annotations for INCEpTION's pod.
+# @section -- INCEpTION deployment settings
+podAnnotations: {}
+
+# -- Extra labels for INCEpTION's pod.
+# @section -- INCEpTION deployment settings
+podLabels: {}
+
+# -- User-defined init-container to run before INCEpTION pod, as defined in [Kubernetes documentation](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/).
+# @section -- INCEpTION deployment settings
+initContainers: {}
+
+# -- User-defined extra containers to run alongside INCEpTION pod, as defined in [Kubernetes documentation](https://kubernetes.io/docs/concepts/workloads/pods/#using-pods).
+# @section -- INCEpTION deployment settings
+extraContainers: {}
+
+# -- Extra environment variables for INCEpTION's container.
+# @section -- INCEpTION deployment settings
+extraEnv: {}
+
+# -- Extra volume definition for INCEpTION's pod.
+# @section -- INCEpTION deployment settings
+extraVolumes: {}
+# @section -- INCEpTION deployment settings
+
+# -- Extra volume mounts for INCEpTION's pod.
+# @section -- INCEpTION deployment settings
+extraVolumeMounts: {}
+
+# INCEpTION container image settings.
+image:
+  # -- Image repository.
+  # @section -- INCEpTION deployment settings
+  repository: "ghcr.io/inception-project/inception"
+  # -- Image Pull Policy.
+  # @section -- INCEpTION deployment settings
+  pullPolicy: "IfNotPresent"
+  # -- Image tag.
+  # @section -- INCEpTION deployment settings
+  tag: "33.3"
+
+# -- INCEpTION container image Pull Secret.
+# @section -- INCEpTION deployment settings
+imagePullSecrets: {}
+
+# INCEpTION's pod startup probe.
+startupProbe:
+  # -- Enables the startup probe.
+  # @section -- INCEpTION deployment settings
+  enabled: true
+  # -- Startup probe initial startup delay.
+  # @section -- INCEpTION deployment settings
+  initialDelaySeconds: 30
+  # -- Startup probe check interval, in seconds.
+  # @section -- INCEpTION deployment settings
+  periodSeconds: 10
+  # -- Startup probe's number of accepted failures.
+  # @section -- INCEpTION deployment settings
+  failureThreshold: 5
+
+# INCEpTION's pod liveness probe.
+livenessProbe:
```

Comment:
INCEpTION has an actuator endpoint that can be queried whether the system is ready. Should that be used here?